Resources/public/js/views/fields/ez-binaryfile-editview.js

```diff
@@ -128,6 +128,23 @@ YUI.add('ez-binaryfile-editview', function (Y) {
         },
 
         /**
+         * Override _showDescription from fieldeditview to do nothing.
```

Comment:
instead of overriding the `_showDescription` and `_hideDesription` to disable the field description visibility handling, I would rather add a check on a property (like `_toggleFieldDefinitionDescription`) in the defaut implementation and just set that property when you don't want the field description to be hidden/shown.